app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1411,7 +1404,7 @@ def rbac_group_validate?
     @assigned_filters = [] if @edit[:new][:filters].empty? || @edit[:new][:use_filter_expression]
     @filter_expression = [] if @edit[:new][:filter_expression].empty? || @edit[:new][:use_filter_expression] == false
     if @edit[:new][:role].nil? || @edit[:new][:role] == ""
-      add_flash(_("A User Group must be assigned a Role"), :error)
+      add_flash(_("A Role must be assigned to a Group"), :error)
```

Comment:
I think this should be "A Group must be assigned a Role".
But if you want to switch it around, "A Role must be assigned to **this** Group", otherwise, it sounds like "fix the role", not "fix the group".

Reply:
For me "A Group must be assigned a Role" does not make sense. I think there's still missing something. So I am better with "A Role must be assigned to *this* Group"

Reply:
I'm fine with both of those :) Thanks :)